torchvision/transforms.py

```diff
@@ -516,6 +516,39 @@ def adjust_gamma(img, gamma, gain=1):
     return img
 
 
+def rotate(img, angle, resample=False, expand=False, center=None, translate=None):
+    """Rotate the image by angle and then (optionally) translate it by (n_columns, n_rows)
+
+
+    Args:
+        img (PIL Image): PIL Image to be rotated.
+        angle ({float, int}): In degrees degrees counter clockwise order.
+        resample ({PIL.Image.NEAREST, PIL.Image.BILINEAR, PIL.Image.BICUBIC}, optional):
+            An optional resampling filter.
+            See http://pillow.readthedocs.io/en/3.4.x/handbook/concepts.html#filters
+            If omitted, or if the image has mode "1" or "P", it is set to PIL.Image.NEAREST.
+        expand (bool, optional): Optional expansion flag.
+            If true, expands the output image to make it large enough to hold the entire rotated image.
+            If false or omitted, make the output image the same size as the input image.
+            Note that the expand flag assumes rotation around the center and no translation.
+        center (2-tuple, optional): Optional center of rotation.
+            Origin is the upper left corner.
+            Default is the center of the image.
+        translate (2-tuple, optional): An optional post-rotate translation.
+    """
+    if not _is_pil_image(img):
+        raise TypeError('img should be PIL Image. Got {}'.format(type(img)))
+
+    if angle < -360 or angle > 360:
+        if angle < 0:
+            angle += 360
+        else:
+            angle -= 360
+        warnings.warn("You could use {} as angle to obtain the same result". format(angle))
```

Comment:
not sure we want to have the warning for this? It could end up cluttering the console